apps/labs/posts/czi-eoss-5-conda-forge.md

```diff
@@ -0,0 +1,280 @@
+      style={{ width: '400px', height: 'auto', objectFit: 'contain' }}
+    />
+    </a>
+    <figcaption>Figure 1e: The former Status page rendered directly on <code>conda-forge/status</code>.</figcaption>
+  </figure>
+  
+  <figure style={{ textAlign: 'center', flex: 1, margin: 0 }}>
+    <a href="/posts/czi-eoss-5-conda-forge/conda-forge-old-6.png" target="_blank">
+    <img 
+      src="/posts/czi-eoss-5-conda-forge/conda-forge-old-6.png"
+      alt="Screenshot of the original Packages page in the conda-forge website, with a search bar that filters which packages are provided by which feedstocks. Text in neon green on white background."
+      style={{ width: '400px', height: 'auto', objectFit: 'contain' }}
+    />
+    </a>
+    <figcaption>Figure 1f: This is how the Packages page looked like in the old site.</figcaption>
+  </figure>
+</div>
+
+One of those add-ons was the `/status` page, which consisted on a periodically regenerated static HTML document templated with Jinja. The idea was to rewrite this page with modern web components, like a React app that could fetch the necessary data on demand, live, without having to resort to cronjob-driven regeneration.
+
+In parallel, we were trying to write some documentation for the infrastructure details we were discovering as part of our audit and research. After all, one of the problems we detected is that the existing documentation was too scarce and out of date. Maybe because the contribution workflow in the old conda-forge website was very clunky. Specially in comparison to what we had observed in other projects that were using [Docusaurus](https://docusaurus.io/). Our hunch was that if we smoothened the contribution frictions, maybe more folks would be inclined to help maintain the documentation.
+
+We decided to start a prototype Docusaurus website just to see how it felt to write for this platform, and... the writing experience was too good to ignore. Live reload just _works_ and it is instantaneous. Docusaurus would not only allow us to write a React app for the status page. It would also enable refactors in all those other Sphinx add-ons, like the blog or the announcements page and RSS feed. Instead of having many loosely coupled repositories, we could have _one_ website built from _one_ place...
+
+So we went for it!
+
+While Afshin Darian (QuantStack) was building the new status page (link to PRs here), we were lucky to have Asmit Malakannawar join us for an internship in conda-forge. He was in charge of designing a new, modern frontpage for the Docusaurus site. You can read about this experience in this blog post. Together with Tania Allard and Isabela Presedo-Floyd (Quansight), as well as Gabriela Vives (QuantStack), we ensured that the new design and color system complied to essential accessibility affordances.
```

Comment:
To do:

- Elaborate a bit more on the status page setup (why it is static, and not Quetz based)
- Add those PR links